CHANGELOG.md

```diff
@@ -1,6 +1,7 @@
 # Changelog
 
 ## [Unreleased]
+- Fixed documentation in boards\atsamv71_xult to refer to probe-rs instead of cargo-embed
```

Comment:
Please refrain from using backspaces in paths, this is very Windowsy, but incorrect.
```suggestion
- Fixed documentation in `boards/atsamv71_xult` to refer to `probe-rs` instead of `cargo-embed`
```